script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector(".todo-form");
+const toDoInput = toDoForm.querySelector("input");
+const toDoInputBtn = document.querySelector(".todo-input").querySelector("div");
+const toDoListPre = document.querySelector(".todo-list-pre");
+const toDoListDone = document.querySelector(".todo-list-done");
+
+let toDoList = [];
+let indPre = 0;
+let indDone = 0;
+
+const createToDo = (event) => {
+  event.preventDefault();
+
+  const toDoObj = {
+    done: false,
+    text: toDoInput.value,
+    id: indPre + 1,
+  };
+
+  drawToDo(toDoObj);
+  toDoInput.value = "";
+  indPre += 1;
+
+  saveListCache();
+};
+
+const drawToDo = (toDoObj) => {
+  const li = document.createElement("li");
+
+  const span = document.createElement("span");
+  span.addEventListener("click", toggleToDo);
+  span.innerHTML = toDoObj.text;
+  li.appendChild(span);
+
+  const delButton = document.createElement("img");
+  delButton.src = "img/ios_trash.png";
+  delButton.addEventListener("click", deleteToDo);
+  li.appendChild(delButton);
+
+  li.id = toDoObj.id;
+
+  if (toDoObj.done === false) {
+    toDoListPre.appendChild(li);
+    document.getElementById("pre").innerHTML =
+      Number(document.getElementById("pre").innerHTML) + 1;
+  } else {
+    toDoListDone.appendChild(li);
+    document.getElementById("done").innerHTML =
+      Number(document.getElementById("pre").innerHTML) + 1;
+  }
+};
+
+const deleteToDo = (event) => {
+  const { target } = event;
```

Comment:
destructing으로 필요한 부분만 분리해주신 점 아주 좋습니다. 다만 `event`라는 파라미터 내에서 `target`만 사용하고 있으니 파라미터를 애초에 이렇게 써 볼 수도 있겠습니다.
```suggestion
const deleteToDo = ({ target }) => {
```

Reply:
파라미터 `event` 내부를 생각하지 않고 코딩했었던 게 부끄러워지네요..ㅠㅠㅠ
너무 소중한 팁 감사드립니다.